adoc/admin-security-certificates.adoc

```diff
@@ -200,6 +206,18 @@ helm install suse/cert-exporter \
     --set customSecret.certs[0].includeKeys="{ca.crt,tls.crt}" \
     --set customSecret.certs[0].annotationSelector="{cert-manager.io/certificate-name}"
 ----
+Or if you have selected the Helm 3 alternative <<helm-tiller-install>>:
++
+[source,bash]
+----
+helm install suse/cert-exporter ${RELEASE_NAME} \
```

Comment:
In Helm3, the name should come before chart.

Reply:
done